notebooks/visualization/matplotlib-intermediate.ipynb

```diff
@@ -0,0 +1,343 @@
+    "                  316.6, 293.9, 291.2, 289.8, 287.1, 285.8, 303.3, 310.])\n",
+    "temps_1000 = np.array([316.0, 316.3, 308.9, 304.0, 302.0, 300.8, 306.2, 309.8,\n",
+    "                       313.5, 313.3, 308.3, 304.9, 301.0, 299.2, 302.6, 309.0,\n",
+    "                       311.8, 304.7, 304.6, 301.8, 300.6, 299.9, 306.3, 311.3])\n",
+    "# Fake dewpoint data to plot\n",
+    "dewpoint = 0.9 * temps\n",
+    "dewpoint_1000 = 0.9 * temps_1000"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## 2. Multiple Plots on One Figure\n",
+    "\n",
+    "The [Matlibplot: Basics]() notebook explained how to create a figure and add a plot to that figure.  It is also possible to add multiple plots to the same figure.  In this step, a figure is first created and then multiple plots are added, each with their own titles, axes, etc."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cell content replaced by load magic replacement.\n",
+    "fig = plt.figure(figsize=(10, 6))\n",
+    "ax = fig.add_subplot(1, 2, 1)\n",
+    "\n",
+    "# Specify how our lines should look\n",
+    "ax.plot(times, temps, color='tab:red', label='Temperature (surface)')\n",
+    "ax.plot(times, temps_1000, color='tab:red', linestyle=':',\n",
+    "        label='Temperature (isobaric level)')\n",
+    "\n",
+    "# Add labels, title, and display settings\n",
+    "ax.set_xlabel('Time')\n",
+    "ax.set_ylabel('Temperature')\n",
+    "ax.set_title('Temperature Forecast')\n",
+    "ax.grid(True)\n",
+    "ax.legend(loc='upper left')\n",
+    "\n",
+    "# Create the second plot\n",
+    "ax2 = fig.add_subplot(1, 2, 2, sharex=ax, sharey=ax)\n",
+    "ax2.plot(times, dewpoint, color='tab:green', label='Dewpoint (surface)')\n",
+    "ax2.plot(times, dewpoint_1000, color='tab:green', linestyle=':', marker='o',\n",
+    "         label='Dewpoint (isobaric level)')\n",
+    "\n",
+    "# Add labels, title, and display settings\n",
+    "ax2.set_xlabel('Time')\n",
+    "ax2.set_ylabel('Dewpoint')\n",
+    "ax2.set_title('Dewpoint Forecast')\n",
+    "ax2.grid(True)\n",
+    "ax2.legend(loc='upper left')\n",
+    "ax2.set_ylim(257, 312)\n",
+    "ax2.set_xlim(95, 162)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## 3. Colorizing Scatter Plots\n",
+    "\n",
+    "Previously, in the [Matlibplot: Basics]() notebook, an example for drawing scatter plots was provided by setting the **linestyle** to non, and adding 'o' markers.  Another alternative is to use the **scatter** methods, while these are slower, they allow more control of the display of the data.  In this case, the data points will be colorized individually based upon a third variable."
```

Comment:
"non" -> "none"

Reply:
Also, maybe "allow more control" -> "allow more visualization options, like varying the color or size of the markers".

Reply:
Added these changes with updated PR